test/web.js

```diff
@@ -616,4 +616,91 @@ lab.experiment("OAuth", function() {
       });
     });
   });
+
+  lab.test("GET /user", function(done) {
+    ls.start(function(error) {
+      var accessTokenRequest = {
+        method: "GET",
+        url: "/user",
+        headers: {
+          "authorization": "token testAccessToken"
+        }
+      };
+
+      s.inject(accessTokenRequest, function(response) {
+        Code.expect(response.statusCode).to.equal(200);
+        Code.expect(response.result.username).to.equal("test");
+        Code.expect(response.result.email).to.equal("test@example.com");
+        done();
+      });
+    });
+  });
+
+  lab.test("GET /user invalid auth", function(done) {
+    ls.start(function(error) {
+      var accessTokenRequest = {
+        method: "GET",
+        url: "/user",
+        headers: {
+          "authorization": "basic nonsenseauthheader"
+        }
+      };
+
+      s.inject(accessTokenRequest, function(response) {
+        Code.expect(response.statusCode).to.equal(401);
+        done();
+      });
+    });
+  });
+
+  lab.test("GET /user invalid auth", function(done) {
+    ls.start(function(error) {
+      var accessTokenRequest = {
+        method: "GET",
+        url: "/user",
+        headers: {
+          "authorization": "token"
+        }
+      };
+
+      s.inject(accessTokenRequest, function(response) {
+        Code.expect(response.statusCode).to.equal(401);
+        done();
+      });
+    });
+  });
+
+  lab.test("GET /user invalid auth", function(done) {
+    ls.start(function(error) {
+      var accessTokenRequest = {
+        method: "GET",
+        url: "/user",
+        headers: {
+          "authorization": "token fakeToken"
+        }
+      };
+
+      s.inject(accessTokenRequest, function(response) {
+        Code.expect(response.statusCode).to.equal(401);
+        done();
+      });
+    });
+  });
+
+  lab.test("GET /user invalid auth", function(done) {
```

Comment:
I like test names w/ descriptive titles b/c they end up being useful as documentation and they make it easier to figure out what test is failing... While I'm not familiar w/ your test framework, it seems like the above 4 tests are all called "GET /user invalid auth". Rather than that I might consider calling them e.g.:

* GET /user returns 401 when authorization header is malformed
* GET /user returns 401 when authorization header has no token
* GET /user returns 401 when authorization header has invalid token
* GET /user returns 401 when authorization header has expired token

That will make it easier for readers to understand what it is the test is trying to verify, and if the tests ever fail, it'll be easier for folks to understand exactly what is amiss from an "end-user" (or in this case an oauth2 client) perspective.